project/CelebornBuild.scala

```diff
@@ -492,30 +492,6 @@ object Spark34 extends SparkClientProjects {
 
   val sparkVersion = "3.4.1"
   val zstdJniVersion = "1.5.2-5"
-
-  lazy val deps = Seq(
-    // Spark Use `log4j-slf4j2-impl` instead of `log4j-slf4j-impl` in SPARK-40511
-    // to fix the error:
-    // ```
-    //   java.lang.NoSuchMethodError: org.apache.logging.slf4j.Log4jLoggerFactory.<init>(Lorg/apache/logging/slf4j/Log4jMarkerFactory;)V
-    // ```
-    "org.apache.logging.log4j" % "log4j-slf4j2-impl" % "2.19.0" % "test"
```

Comment:
it's a transitive dep of spark-core, have you explicitly excluded it?

Reply:
it is not found in the current dependency tree.

```
$ ./build/sbt -Pspark-3.4 celeborn-client-spark-3/Test/dependencyTree | grep log4j-slf4j
[info]   | | +-org.apache.logging.log4j:log4j-slf4j-impl:2.17.2
[info]   | +-org.apache.logging.log4j:log4j-slf4j-impl:2.17.2
[info]   | +-org.apache.logging.log4j:log4j-slf4j-impl:2.17.2
[info]   | | | +-org.apache.logging.log4j:log4j-slf4j-impl:2.17.2
[info]   | | +-org.apache.logging.log4j:log4j-slf4j-impl:2.17.2
[info]   |   +-org.apache.logging.log4j:log4j-slf4j-impl:2.17.2
```